app/src/main/java/itaycsguy/rtchordslearningapk/SigninActivity.kt

```diff
@@ -1,10 +1,10 @@
 package itaycsguy.rtchordslearningapk
 
-import android.app.ProgressDialog.show
 import android.content.Intent
+import android.net.wifi.p2p.WifiP2pDevice.CONNECTED
 import android.support.v7.app.AppCompatActivity
 import android.os.Bundle
-import android.support.constraint.ConstraintLayout
+import android.support.v4.content.ContextCompat.startActivity
```

Comment:
check if u can move to v7 since we use it on other imports and we wont to be consistent

Reply:
done!